addons/pkg/constants/constants.go

```diff
@@ -126,6 +126,24 @@ const (
 
 	// InfrastructureRefDocker is the docker infrastructure
 	InfrastructureRefDocker = "DockerCluster"
+
+	// DefaultCNISelectionClusterVariableName is the default cluster variable name for cni selection
+	DefaultCNISelectionClusterVariableName = "tkg.tanzu.vmware.com/cni"
+
+	// DefaultHTTPProxyClusterClassVarName is the default cluster variable name for HTTP proxy setting
```

Comment:
Checked with Nick, they are using the var name they had directly. Not sure if we need to add `tkg.tanzu.vmware.com/` prefix here

Reply:
Adding tkg.tanzu.vmware.com/ prefix for now. Need to sync with CLI team to finalize the naming convention.
cc @tenczar